src/tests/regress/data/sql/float4.sql

```diff
@@ -349,6 +349,6 @@ SELECT '-9223380000000000000'::float4::int8;
 --@   from (select bits::integer::xfloat4::float4 as flt
 --@           from testdata
 --@ 	offset 0) s;
---@ 
---@ -- clean up, lest opr_sanity complain
```

Comment:
This was actually leveraging another bug that we did not treat `--@ \n` as an empty line, and there is no corresponding line in expected output file. After we fixed both issues in this PR. either left or right could be correctly broken up into individual test cases.